tests/test_select.rs

```diff
@@ -104,6 +104,11 @@ select_test!(select_range_no_end_cat, "h4-,h1", "4-,1",
 select_test!(select_range_no_start_cat, "-h2,h1[1]", "-2,5",
              ["h1", "h2", "h1"], ["a", "b", "e"]);
 
+select_test!(select_regex_double, r#"/h[1-3]/"#, "1,2,5",
+             ["h1", "h2", "h1"], ["a", "b", "e"]);
+select_test!(select_regex_single, r#"/h[1-3]/"#, "1,2,5",
```

Comment:
I've removed the duplicate test and added test cases for these examples. I also added a check for an empty regex, on the assumption that the expected behaviour for `//` is to fail (since it's an empty regex). I'm a bit unsure about the test for `/`; perhaps we should have a different test CSV with a `/` in the headers and ensure it's still selected.